### DISCUSSION TITLE
[UI/UX] Search input for profile language and currency

### DISCUSSION THREAD

**Discussion by Phyloiota** [iotaledger/firefly#1570] [Features / Enhancements]
## Description

Would make changing the profile language and currency much easier if i could type in search phrases. So instead of scrolling up and down to look for a language i could just type in "F" and would get all languages that start with an F, etc etc
Same for currency - search for "U" will bring up USD much faster than scrolling through all available currencys.

## Motivation

Ease of use

## Requirements

implement a search tool in the dropdown menu for currency and language
<img width="363" alt="currency search" src="https://user-images.githubusercontent.com/77154511/114260905-2d94a980-9a0a-11eb-89db-21cbd640c85a.png">
<img width="399" alt="language search" src="https://user-images.githubusercontent.com/77154511/114260906-2ff70380-9a0a-11eb-9f37-0295189c4d9c.png">

**Comment by miili** [comment]
Also consider adding `IOTA` to the main screen widget.

![image](https://user-images.githubusercontent.com/4992805/115020628-26aee080-9ebb-11eb-85f8-b6ade441813b.png)

**Comment by maxwellmattryan** [comment]
Hey @Phyloiota, I'm closing this issue and moving it to a discussion [here](https://github.com/iotaledger/firefly/discussions/1502) 👍
